src/blocks/base_block.rs

```diff
@@ -54,10 +57,12 @@ impl<T: Block> Block for BaseBlock<T> {
 pub(super) struct BaseBlockConfig {
     /// Command to execute when the button is clicked
     pub on_click: Option<String>,
+
+    pub color_overrides: Option<HashMap<String, String>>,
 }
 
 impl BaseBlockConfig {
-    const FIELDS: &'static [&'static str] = &["on_click"];
+    const FIELDS: &'static [&'static str] = &["on_click", "color_overrides"];
```

Comment:
Let's just change this to `theme_overrides`.

Reply:
Yeah, I also thought that would be a better name.